### ISSUE TITLE
Knightmove fails

### ISSUE THREAD

**Issue by yo35** [yo35/rpb-chessboard#134]
Reported by bonaventura61:

> Try this pgn:

```
[Event “Freie Partie”]
[Site “?”]
[Date “1949.01.10”]
[Round “?”]
[White “Schmidt, Alice”]
[Black “Schmidt, Arno”]
[Result “0-1”]
[ECO “C37”]
[PlyCount “82”]
[EventDate “1949.??.??”]

1. e4 e5 2. f4 exf4 3. Nf3 g5 4. d4 g4 5. Bc4 gxf3 6. Qxf3 Bh6 7. Bxf4 Bxf4 8.
Qxf4 f6 9. O-O d6 10. e5 f5 11. Be2 h5 12. Qg3 Qe7 13. exd6 Qxe2 14. Re1 Qxe1+
15. Qxe1+ Kd7 16. Qe5 Rh6 17. dxc7 Nc6 18. Qg7+ Nce7 19. Nc3 Rd6 20. Re1 b6 21.
Nb5 Rd5 22. c4 Rxb5 23. cxb5 Bb7 24. Re5 Be4 25. Qh7 Rc8 26. Qxh5 Rxc7 27. g4
Rc1+ 28. Kf2 Rc2+ 29. Kg3 Nf6 30. Qf7 Rg2+ 31. Kh3 fxg4+ 32. Kh4 Rxh2+ 33. Kg3
Rh3+ 34. Kf4 Rf3+ 35. Kg5 Nh7+ 36. Qxh7 Bxh7 37. Kxg4 Rf2 38. d5 Rxb2 39. Rh5
Bf5+ 40. Kf4 Rxb5 41. Ke5 Rxd5+ 0-1
```
> Move 18 by Black is not correctly processed by your plugin version 5.3.2.

**Comment by yo35** [comment]
It seems this regression has appeared with version 5.3.

**Comment by yo35** [comment]
Fixed in version 5.3.3 (just released).
